src/instana/options.py

```diff
@@ -13,57 +13,79 @@
     - AWSFargateOptions - Options class for AWS Fargate.  Holds settings specific to AWS Fargate.
     - GCROptions - Options class for Google cloud Run.  Holds settings specific to GCR.
 """
+
 import os
 import logging
+from typing import Any, Dict
+
+from instana.collector.helpers.runtime import parse_ignored_endpoints
```

Comment:
As commented above, the `parse_ignored_endpoints` function should be imported from `src/instana/util/runtime.py` or, even better, from `src/instana/util/config.py`.